.github/workflows/go.yml

```diff
@@ -28,7 +28,7 @@ jobs:
 
     - name: Build
       run: |
-        make cross qemu-wrapper vm win-gvproxy win-sshproxy image
+        make cross qemu-wrapper vm win-gvproxy win-sshproxy
```

Comment:
I think we could build the image with each PRs to know when there are regressions building it, but only push it for releases.